docs/guide/src/privacy.md

```diff
@@ -0,0 +1,31 @@
+# Privacy
+
+This document describes the privacy features of the Penumbra protocol, as well as conditions that must be true for privacy to be preserved.
+
+## Transfers
+
+Transfers do not reveal the asset, amount, sender or recipient identity. Transfers are also unlinkable to each other.
+
+## Swaps
+
+Initiating a swap does not reveal the identity of the swapper or the pre-paid claim fee, but it does reveal the assets and amounts in the swap.
+
+Claiming a swap does not reveal the amounts, asset types or the identity of the claimant.
```

Comment:
While the two notes that get minted in a successful `SwapClaim` are private (including amounts/asset types), the `SwapClaim` action does reveal the _trading pair_ due to the claimant having to provide the public `BatchSwapOutputData` which has the trading pair as a field. 

The shielded output notes are not linkable to the `BatchSwapOutputData` though, so an observer of the chain just learns an anonymous someone minted shielded outputs of a swap of assets A/B from a swap that was executed in block X.

Reply:
Thank you - I've tried to communicate this better in the updated version.